internal/cmd/cmd.go

```diff
@@ -84,7 +84,7 @@ func doLint(
 
 	flags, err := lint.NewFlags(args)
 	if err != nil {
-		_, _ = fmt.Fprint(stderr, err)
+		_, _ = fmt.Fprintln(stderr, err)
```

Comment:
[memo] It enables it to properly show an error message when `err` ends up without a new line.

```
❯❯❯ ./protolint -plugin ./not_existence _example/proto/
failed client.Client(), err=fork/exec ./plugin/greete: no such file or directory
```